spec/support/endpoint_expectations.rb

```diff
@@ -0,0 +1,37 @@
+module EndpointExpectations
+  def expect_api_request_is_correct(
```

Comment:
At this point, I would look into using a custom matcher (As in RSpec custom matcher). On thought is you could change `stub_*` to return an `api` for use in units. So something like:

```ruby
subject = described_class.new(id: 'group1')
path =  'ticket_groups/group1'

stub_delete(path: path)

expect(subject.delete).to eq(true)
expect_api_delete_is_correct(path: path)
```

could change to:

```ruby
subject = described_class.new(id: 'group1')
path =  'ticket_groups/group1'

api = stub_delete(path: path)

expect(subject.delete).to eq(true)
expect(api).to called(:delete).with(path: path)
```